src/runner/pythonrunner.cpp

```diff
@@ -251,33 +262,36 @@ BOOST_PYTHON_MODULE(mobase)
       .def("basePath", &IOrganizer::basePath)
       .def("modsPath", &IOrganizer::modsPath)
       .def("appVersion", &IOrganizer::appVersion)
-      .def("getMod", &IOrganizer::getMod, bpy::return_value_policy<bpy::reference_existing_object>())
-      .def("createMod", &IOrganizer::createMod, bpy::return_value_policy<bpy::reference_existing_object>())
-      .def("getGame", &IOrganizer::getGame, bpy::return_value_policy<bpy::reference_existing_object>())
-      .def("removeMod", &IOrganizer::removeMod)
-      .def("modDataChanged", &IOrganizer::modDataChanged)
-      .def("pluginSetting", &IOrganizer::pluginSetting)
-      .def("setPluginSetting", &IOrganizer::setPluginSetting)
-      .def("persistent", &IOrganizer::persistent, bpy::arg("persistent") = QVariant())
-      .def("setPersistent", &IOrganizer::setPersistent, bpy::arg("sync") = true)
+      .def("getMod", &IOrganizer::getMod, bpy::return_value_policy<bpy::reference_existing_object>(), bpy::arg("name"))
+      .def("createMod", &IOrganizer::createMod, bpy::return_value_policy<bpy::reference_existing_object>(), bpy::arg("name"))
+      .def("getGame", &IOrganizer::getGame, bpy::return_value_policy<bpy::reference_existing_object>(), bpy::arg("name"))
+      .def("removeMod", &IOrganizer::removeMod, bpy::arg("mod"))
+      .def("modDataChanged", &IOrganizer::modDataChanged, bpy::arg("mod"))
+      .def("pluginSetting", &IOrganizer::pluginSetting, (bpy::arg("plugin_name"), "key"))
+      .def("setPluginSetting", &IOrganizer::setPluginSetting, (bpy::arg("plugin_name"), "key", "value"))
+      .def("persistent", &IOrganizer::persistent, (bpy::arg("plugin_name"), "key", bpy::arg("persistent") = QVariant()))
+      .def("setPersistent", &IOrganizer::setPersistent, (bpy::arg("plugin_name"), "key", "value", bpy::arg("sync") = true))
       .def("pluginDataPath", &IOrganizer::pluginDataPath)
-      .def("installMod", &IOrganizer::installMod, (bpy::arg("name_suggestion") = ""), bpy::return_value_policy<bpy::reference_existing_object>())
```

Comment:
The order's changed here. Is it intentional?

Reply:
Yes, The way it's implement in `boost::python` makes it so the order is irrelevant... I tried to be consistent and always put argument names at the end (which was already the case for the ones that were already there) so I changed this.